test/extended/etcd/etcd_storage_path.go

```diff
@@ -74,7 +75,7 @@ func GetOpenshiftEtcdStorageData(namespace string) map[schema.GroupVersionResour
 
 		// github.com/openshift/api/apps/v1
 		gvr("apps.openshift.io", "v1", "deploymentconfigs"): {
-			Stub:             `{"metadata": {"name": "dc1g"}, "spec": {"selector": {"d": "c"}, "template": {"metadata": {"labels": {"d": "c"}}, "spec": {"containers": [{"image": "image-registry.openshift-image-registry.svc:5000/openshift/tools:latest", "name": "container2"}]}}}}`,
+			Stub:             `{"metadata": {"name": "dc1g"}, "spec": {"selector": {"d": "c"}, "template": {"metadata": {"labels": {"d": "c"}}, "spec": {"containers": [{"image": extutilimage.ShellImage(), "name": "container2"}]}}}}`,
```

Comment:
We'll need to `Sprintf()` or concat this call too. 

Also, small typo in the package call that's present in the over files.

Reply:
Thanks!  Fixed with 8c6746d -> bf132bc.